Dev/Homebrew/package_manager.7h.py

```diff
@@ -93,12 +110,12 @@ def active(self):
     def run(self, *args):
         """ Run a shell command, return the output and keep error message.
         """
-        self.error = None
```

Comment:
Pretty sure you want to leave this as nothing else clears the error state. So without this don't you end up with a single error never going away? (Mobile so maybe I'm missing a similar line somewhere else.)